<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DormFlows</class>
 <widget class="QDialog" name="DormFlows">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>473</width>
    <height>173</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DormFlows</string>
  </property>
  <property name="windowIcon">
   <iconset resource="dormflows.qrc">
    <normaloff>:/img/flow.png</normaloff>:/img/flow.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="imgL">
       <property name="text">
        <string>img                            </string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QProgressBar" name="flowsProg">
         <property name="maximum">
          <number>8300</number>
         </property>
         <property name="value">
          <number>0</number>
         </property>
         <property name="format">
          <string>%vM/%mM</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="flowsL">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QCheckBox" name="disableChk">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="limitLE">
           <property name="inputMask">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="disL">
           <property name="text">
            <string>M時停用&quot;eth0&quot;</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>IP:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="ipLE">
           <property name="inputMask">
            <string>000.000.000.000; </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="text">
            <string>重啟</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="dormflows.qrc"/>
 </resources>
 <connections/>
</ui>
